src/ncp-spinel/SpinelNCPControlInterface.h

```diff
@@ -163,6 +163,23 @@ class SpinelNCPControlInterface : public NCPControlInterface, public NCPMfgInter
 		CallbackWithStatus cb = NilReturn()
 	);
 
+	virtual void add_service(
+		uint32_t enterprise_number, 
+		const uint8_t *service_data, 
+		unsigned int service_data_len, 
```

Comment:
Suggestion: How about using `Data` type for parameters service and server data?  `const Data &service_data`?

Reply:
Done